plugins/chat/plugin.rb

```diff
@@ -37,6 +37,7 @@ module ::Chat
     chat_channel_retention_days: :dismissed_channel_retention_reminder,
     chat_dm_retention_days: :dismissed_dm_retention_reminder,
   }
+  PRESENCE_REGEXP = %r{^/chat-reply/(\d+)(?:/thread/(\d+))?$}
```

Comment:
Extracted this regular expression here because it felt wasteful to re-create it all the time.

I would have liked to extract it "closer" to where it's used, but we have rubocop rules that prevents us from declaring constants outside a module/class.